src/libm/CMakeLists.txt

```diff
@@ -1,31 +1,66 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/include/)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
 # Helper executable: generates parts of the sleef header file
 add_executable(${TARGET_MKRENAME} mkrename.c)
-target_compile_options(${TARGET_MKRENAME}
-  PRIVATE ${FLAGS_WALL})
 target_compile_features(${TARGET_MKRENAME}
   PUBLIC c_variadic_macros)
+# Helper executable: dispatcher for the vector extensions
+add_executable(${TARGET_MKDISP} mkdisp.c)
+target_compile_features(${TARGET_MKDISP}
+  PUBLIC c_variadic_macros)
+
+set(RENAME_HEADERS "")
+set(TARGET_OBJECTS "")
+
+# Single precision and double precision
+set(SIMD_SOURCES sleefsimdsp.c sleefsimddp.c)
+
+if(COMPILER_SUPPORTS_SSE2)
+  set(HEADER_SSE2 ${CMAKE_CURRENT_BINARY_DIR}/include/renamesse2.h)
+  set(OBJECT_SSE2 "sleefsse2")
+
+  add_library(${OBJECT_SSE2} OBJECT ${SIMD_SOURCES} ${HEADER_SSE2})
```

Comment:
I have found a more portable way to tell cmake I would like to compile position independent code. See next patch.